smartmontools/drivedb.h

```diff
@@ -4557,6 +4557,13 @@ const drive_settings builtin_knowndrives[] = {
     "", "",
     "-v 9,msec24hour32 -v 188,raw16 -v 240,msec24hour32"
   },
+  { "Seagate Enterprise Capacity 3.5 HDD", // untested
+    "ST12000NM0127-.*",
```

Comment:
@OddMagnet, I tested it and It's not working like this. It should be just `ST12000NM0127`. You can put also "Standard 512E model". The regex should be `ST12000NM01[12]7`

Reply:
Thanks, I've updated the regex. Can you confirm it's working with your changes? Then I'd remove the `//untested` as well

Reply:
I tested it. It works as expected.